ui/src/router/components/form/components/ensembler_config/RouteSelectionPanel.js

```diff
@@ -1,48 +1,78 @@
 import React, { useEffect } from "react";
-import { EuiFlexItem, EuiForm, EuiFormRow } from "@elastic/eui";
+import { EuiFlexItem, EuiForm, EuiFormRow, EuiSuperSelect } from "@elastic/eui";
 
+import { RouteDropDownOption } from "../RouteDropDownOption";
 import { Panel } from "../Panel";
-import { EuiComboBoxSelect } from "../../../../../components/form/combo_box/EuiComboBoxSelect";
 
 const SelectRouteIdCombobox = ({ value, routeOptions, onChange, ...props }) => {
-  // Note: routeNames changes on every render, which we need to capture name changes to routes
-  const routeNames = routeOptions.map((e) => (!!e.value ? e.value : e.label));
+  const noneValue = "_none_";
 
   useEffect(() => {
-    // Clear route if it doesn't exist in the options
-    if (!!value && !routeNames.includes(value)) {
-      onChange(undefined);
+    const routeNames = routeOptions
+      .filter((e) => !e.disabled)
+      .map((e) => e.value);
+    // Clear route if it doesn't exist in the enabled options
+    if (value !== noneValue && !routeNames.includes(value)) {
+      onChange(noneValue);
     }
     // Auto-select route if there is exactly one
-    if (!value && routeNames.length === 1) {
+    if ((!value || value === noneValue) && routeNames.length === 1) {
       onChange(routeNames[0]);
     }
-  }, [routeNames, value, onChange]);
+  }, [routeOptions, value, onChange]);
+
+  const routeOptionsWithNone = [
+    // Hack to have a placeholder for EuiSuperSelect
+    {
+      value: noneValue,
+      inputDisplay: props.placeholder,
+      disabled: true,
+    },
+    ...routeOptions,
+  ];
 
   return (
-    <EuiComboBoxSelect
+    <EuiSuperSelect
```

Comment:
Changing to `EuiSuperSelect` to have a richer dropdown, similar to the traffic rules panel.